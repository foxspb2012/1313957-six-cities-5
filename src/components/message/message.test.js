import React from 'react';
import Message from './message';
import renderer from 'react-test-renderer';

describe(`Should Message render correctly`, () => {
  it(`Should Message render correctly with text`, () => {
    const tree = renderer
      .create(
          <Message
            title={`Text`}
          />
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
