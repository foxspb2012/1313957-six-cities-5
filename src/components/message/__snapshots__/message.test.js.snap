// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Message render correctly Should Message render correctly with text 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "color": "#ff0000",
      "display": "flex",
      "flexDirection": "column",
      "justifyContent": "flex-start",
      "margin": "20px",
      "padding": "20px",
      "textAlign": "center",
      "width": "100%",
    }
  }
>
  <h2
    style={
      Object {
        "fontSize": "24px",
        "fontStyle": "oblique",
        "fontWeight": 700,
        "lineHeight": 1.1667,
      }
    }
  >
    Attention
  </h2>
  <p>
    Text
  </p>
</div>
`;
